tasks/verify-conforma-konflux-ta/0.1/verify-conforma-konflux-ta.yaml

```diff
@@ -336,6 +330,22 @@ spec:
         - '{policy: .policy, key: .key, "effective-time": .["effective-time"]}'
         - "$(params.HOMEDIR)/report-json.json"
 
+    - name: detailed-report
+      image: quay.io/conforma/cli:latest
+      command: ["sh", "-c"]
+      args:
+        - |
+          # Check if the report file exists before trying to display it
+          if [ -f "$(params.HOMEDIR)/detailed-text-report.txt" ]; then
+            cat "$(params.HOMEDIR)/detailed-text-report.txt" || true
+          else
+            echo "Warning: detailed-text-report.txt not found. Ensure validate step completed successfully and produced the file."
+          fi
+      volumeMounts:
+        - mountPath: /var/workdir
+          name: workdir
```

Comment:
What's this for? I don't see this for the other steps, so I expect we don't need it here.

Update: See `stepTemplate` above, I guess all steps have this volumeMount added via the stepTemplate, so no need add it here.